tests/jballerina-unit-test/src/test/resources/test-src/runtime/api/values/modules/errors/errors.bal

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2023 WSO2 LLC.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
```

Comment:
``` suggestion
// Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
//
// WSO2 LLC. licenses this file to you under the Apache License,
// Version 2.0 (the "License"); you may not use this file except
// in compliance with the License.
```

Reply:
We don't add this part now right?

Reply:
The license header for the open-source repos should not include "All Rights Reserved" part.
Can refer to the "Re: License Header for WSO2 code" mail.